schema/gendummydata.py

```diff
@@ -41,7 +53,19 @@
 RANDOM_TLDS = ("edu", "com", "org", "net", "tw", "ru", "pl", "de", "es")
 RANDOM_URL = ("http://www.", "ftp://ftp.", "http://", "ftp://")
 RANDOM_LOCS = ("pub", "release", "files", "downloads", "src")
-FORTUNE_FILE = os.environ.get("FORTUNE_FILE", "/usr/share/fortune/cookie")
+
+if (os.name == 'posix'):
+    OS = getosname()
+    if OS in 'Arch Linux':
+        FORTUNE_FILE = os.environ.get("FORTUNE_FILE", '/usr/share/fortune/cookie')
+    elif OS in 'Debian':
+        FORTUNE_FILE = os.environ.get("FORTUNE_FILE", '/usr/share/games/fortunes/fortune')
+    elif OS in 'Ubuntu':
+        # Needs to be verified.
+        FORTUNE_FILE = os.environ.get("FORTUNE_FILE", '/usr/share/games/fortunes/fortune')
```

Comment:
Could you make it something like this instead?

```python3
    if OS in 'Arch Linux':
        FORTUNE_FILE = "/usr/share/fortune/cookie"
    elif OS in 'Debian':
        FORTUNE_FILE = "/usr/share/games/fortunes/fortune"
    elif OS in 'Ubuntu':
        FORTUNE_FILE = "/usr/share/games/fortunes/fortune"
    
    FORTUNE_FILE = os.environ.get("FORTUNE_FILE", FORTUNE_FILE)
```

I think it makes the code cleaner as we only call the `os.environ` stuff at the very end.

---

I need to write it up, but for the time being just note that I prefer to use double quotes over single quotes, and only using the latter when it helps make things more readable.